### PR TITLE
draft of web ta

### PR DESCRIPTION
**Pull request by c2nelson**
draft of guidance on web accessibility

### CODE REVIEW

**Comment by vwilliamsn** [comment]
@c2nelson Looks good on the beta site, link seems to be broken.

File: _includes/news-item.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
         </time>
       </p>
       <h3 class="usa-collection__heading">
-        <a class="usa-link" href="{{ news-item.permalink | relative}}"
+        <a class="usa-link" href="{{ news-item.permalink | relative }}"
```
Comment (JackRyan1989) [inline _includes/news-item.html:14]: @c2nelson You don't need that extra space after 'relative'.

File: _pages/web-ta.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+- **Lack of text alternatives (“alt text”) on images.**  People who are blind will not be able to understand the content and purpose of images, such as pictures, illustrations, and charts, when no text alternative is provided.  Text alternatives convey the purpose of an image, including pictures, illustrations, charts, etc.  
+
+- **No captions on videos.**  People with hearing disabilities may not be able to understand information communicated in a video if the video does not have captions.
+
+- **Inaccessible online forms.**  People with disabilities may not be able to fill out, understand, and accurately submit forms without things like:
+    - Labels that screen readers can convey to their users (such as text that reads “credit card number” where that number should be entered);
```
Comment (JackRyan1989) [inline _pages/web-ta.md:33]: @c2nelson The Markdown linter is reporting that lines 32, 33 & 34 are indented more than expected. Change the indentation from 4 spaces to 2 spaces and re-run.

**Review by JackRyan1989 — COMMENTED** [review]
@c2nelson Please see my comments within the code. I think all you need to do is remove the extra indentation in the MD file in lines 33, 34, &35 and the linter should pass.

**Review by JackRyan1989 — APPROVED** [review]
Looks good to me!

**Comment by c2nelson** [comment]
Thanks-a-million, Jack!!  We're not ready to go yet but at least we passed the lint code base check.  Thanks again!!!

File: _pages/web-guidance.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+A website with inaccessible features can limit the ability of people with disabilities to access a public accommodation’s goods, services, and privileges available through that website—for example, a veterans’ service organization event registration form.  
+
+**For these reasons, the Department has consistently taken the position that the ADA’s requirements apply to all the goods, services, privileges, or activities offered by public accommodations, including those offered on the web.**  
+
+## How to Make Web Content Accessible to People with Disabilities  
+
+**Businesses and state and local governments have flexibility in how they comply with the ADA’s general requirements of nondiscrimination and effective communication.  But they must comply with the ADA’s requirements.**  
+
+The Department of Justice does not have a regulation setting out detailed standards, but the Department’s longstanding interpretation of the general nondiscrimination and effective communication provisions applies to web accessibility.[^1]
```
Comment (JackRyan1989) [inline _pages/web-guidance.md:78]: @c2nelson Replace [^1] with:
<sup id="fnref:1" role="doc-noteref"><a href="#fn:1" class="footnote" rel="footnote">1</a></sup>

File: _pages/web-guidance.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+- [Digital.gov](https://digital.gov/topics/accessibility/): this site, which is part of the Technology Transformation Services at the GSA, has resources on design of products, devices, services, or environments for people with disabilities.  
+- [Section 508 Information and Communication Technology Accessibility Standards](https://www.access-board.gov/ict/): standards published by the U.S. Access Board addressing access to information and communication technology under Section 508 of the Rehabilitation Act of 1973.  
+- [Section508.gov](https://www.section508.gov/): a website published by the GSA with tools and training on implementing website accessibility requirements under Section 508.  
+- [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/wai/standards-guidelines/wcag/): guidelines published by the Web Accessibility Initiative of the World Wide Web Consortium.  
+
+[^1]: <i>See</i> 42 U.S.C. &sect;&sect; 12132, 12182(a); 28 C.F.R. &sect;&sect; 35.130, 35.160(a), 36.201, 36.303(c).
```
Comment (JackRyan1989) [inline _pages/web-guidance.md:129]: Replace line with: 
`<div class="footnotes" role="doc-endnotes">
  <ol>
    <li id="fn:1" role="doc-endnote">
      <p><i>See</i> 42 U.S.C. §§ 12132, 12182(a); 28 C.F.R. §§ 35.130, 35.160(a), 36.201, 36.303(c).&nbsp;<a href="#fnref:1" class="reversefootnote" role="doc-backlink">1</a></p>
    </li>
  </ol>
</div>`

**Review by JackRyan1989 — APPROVED** [review]

**Review by vwilliamsn — APPROVED** [review]
The widget/document looks really good!  Great job @c2nelson, thanks @JackRyan1989 for helping out!!!  Great job team!!!

**Pull request merged**